docs/features/members/overview.md

```diff
@@ -3,20 +3,49 @@ id: members-overview
 title: Overview
 ---
 
-:::caution
+## Overall Guidance
 
-This section is a work in progress.
+This section aims to give an outline of the features that make up the core of how members operate within Talawa and by extension the organisation. We will cover features such as:
+
+1. Member Registration
+2. Member Roles
+3. Validating users for the organization
+4. Access Banning 
+5. Access Recovering
+
+## Member Registration
+This feature requires the creation and implementation of a unique identifier for each member of the organization. This identifier will allow for accurate tracking of each member's activity within the app and enable personalized user experiences. It is imperative to ensure that the identification process is secure and that user privacy is maintained. By having a unique identifier for each user, it will be easier to analyze user behavior, provide better customer support, and identify and address any issues that may arise. [Click here to know more](https://docs.talawa.io/docs/features/members/members-recovery)
+
+## Member Roles
```

Comment:
Can you review this page and align your wording here. (https://docs.talawa.io/docs/features/members/members-roles)  I think for the overview it would be good to keep the descriptions basic, and leave the details of future ideas to the individual pages.  Also, for the links to pages, can you use the relative link instead of the full link?